lib/services/android-project-service.ts

```diff
@@ -118,19 +120,134 @@ class AndroidProjectService implements IPlatformProjectService {
 			return assetsDirectory;
 
 		}).future<string>()();
-	}
+    }
+
+    private updateMetadata(projectRoot: string): void {
+        var projMetadataDir = path.join(projectRoot, "assets", "metadata");
+        var libsmetadataDir = path.join(projectRoot, "../../lib", this.platformData.normalizedPlatformName, "__metadata");
```

Comment:
you can extract __metadata as a constant (due to typescript limitations, a private static member) because it is shared among methods in the class.